lib/netflix2.js

```diff
@@ -205,6 +205,59 @@ Netflix.prototype.__getViewingHistory = function (page, callback) {
   })
 }
 
+/**
+ * Hides viewing history for a specific movie or series
+ * @param movieID  - the ID of the movie (e.g. 80057281 for "Stranger Things")
+ * @param seriesAll - true if you want to clear the whole series, false otherwise
+ */
+Netflix.prototype.hideMovieViewingHistory = function (movieID, seriesAll, callback) {
+  this.__hideMovieViewingHistory(movieID, seriesAll, callback);
+}
+
+Netflix.prototype.__hideMovieViewingHistory = function (movieID, seriesAll, callback) {
+  var options = {
+    body: {
+      movieID: movieID,
+      seriesAll: seriesAll,
+      authURL: this.authUrls[constants.yourAccountUrl]
+    },
+    method: 'POST'
+  }
+  var endpoint = constants.viewingActivity
+  this.__apiRequest(endpoint, options, function (error, response, json) {
+    if (error) {
+      return callback(error)
+    }
+    callback(null, json)
+  })
+}
+
+/**
+ * Hides ALL viewing history: this may not always reset the viewing history per series
```

Comment:
Please add a period at the end of the line to clarify that the next line containt a completely separate sentence.

Also: how can this function not remove the entire viewing history of a series? Do you have an explenation for that or is it just something you observed during development?

Reply:
Added the period.

Passing `seriesAll:false` to the request will remove just the current movieID (e.g. if it's an episode part of series, it will clear just that episode, not the whole series that episode may be part of). Passing `seriesAll:true` will clear the whole series the `movieID` is part of, but not other series of the same show.